cmd/observe/io_reader_observer.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2021 Authors of Hubble
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package observe
+
+import (
+	"bufio"
+	"context"
+	"io"
+
+	"github.com/cilium/cilium/api/v1/observer"
+	v1 "github.com/cilium/cilium/pkg/hubble/api/v1"
+	"github.com/cilium/cilium/pkg/hubble/filters"
+	"github.com/cilium/hubble/pkg/logger"
+	"github.com/golang/protobuf/jsonpb"
```

Comment:
Package `github.com/golang/protobuf` is superseded by `google.golang.org/protobuf`. Please, use `google.golang.org/protobuf/encoding/protojson` to avoid importing `github.com/golang/protobuf` altogether.